EditorWindow/MainGraphicsView.cpp

```diff
@@ -1,5 +1,6 @@
 ﻿#include "MainGraphicsView.h"
 #include "WL4EditorWindow.h"
+#include "Dialog/DoorConfigDialog.h"
```

Comment:
Is this extra include necessary?

Reply:
DoorConfigDialog instances are created in several places in the cpp, it was in the include chain of WL4EditorWindow.h so it was not there before. i think it will be prehaps better to include that explicit ly

Reply:
discussion result: gsb rejected the changes.